V0.2.0 (#73)
ropensci-org/contributing-guidance: rsrc_statsreview.Rmd

Learn about this work in progress at [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/index.html). 

Until recently, rOpenSci's software peer review program  focused on R packages for reproducible data life cycle management
To apply these processes to software implementing statistical methods and algorithms, we need new ways of evaluating and testing software, and of managing the review process, and to bring in new expert editors and reviewers
This book serves as a home for collecting research and developing those standards, documenting not only our guidelines for statistical software but also the process of expanding the scope of review so the general processes of software review may be adapted and reproduced in other domains.

This work is [supported by the Sloan Foundation](https://ropensci.org/blog/2019/07/15/expanding-software-review/) and is organized under an R Consortium Working Group.

### How to contribute

*   **Read our in-progress book** [rOpenSci Statistical Software Peer Review](https://stats-devguide.ropensci.org/index.html) to familiarize yourself with standards and processes under development
*   **Follow the discussion**. 

>     Browse the dedicated [area for discussion on the rOpenSci forum](https://discuss.ropensci.org/c/statistical-software-peer-review/28) or in the #stats-peer-review [Slack](#channels-slack) channel and share your comments or questions.

Reviewer: ```suggestion
    Browse the dedicated [GitHub discussions area](https://github.com/ropensci/statistical-software-review-book/discussions) or in the #stats-peer-review [Slack](#channels-slack) channel and share your comments or questions.
```

should we omit slack for more transparency of such convos?
